lib/ts/recipe/emailpassword/recipeImplementation.ts

```diff
@@ -1,8 +1,12 @@
-import { RecipeInterface, User } from "./types";
+import { RecipeInterface, TypeInputFormField, User } from "./types";
 import { Querier } from "../../querier";
 import NormalisedURLPath from "../../normalisedURLPath";
+import { FORM_FIELD_PASSWORD_ID } from "./constants";
 
-export default function getRecipeInterface(querier: Querier): RecipeInterface {
+export default function getRecipeInterface(
+    querier: Querier,
+    formFields: TypeInputFormField[] | undefined
```

Comment:
Instead of taking formFields, give it a function like `getEmailPasswordConfig: () => NormalisedConfig`